examples/BuddyMatmul/makefile

```diff
@@ -35,3 +35,22 @@ linalg-batchmatmul-f32-run:
 		-reconcile-unrealized-casts | \
 	${MLIR_CPU_RUNNER} ${OPT_FLAG} -e main -entry-point-result=void \
 		-shared-libs=${MLIR_RUNNER_UTILS} -shared-libs=${MLIR_C_RUNNER_UTILS}
+
+linalg-transposematmulb-f32-run:
```

Comment:
Split `transpose`, `matmul`, and `b`. (`transposematmulb` -> `matmul-transpose-b`)